visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.tsx

```diff
@@ -353,8 +397,9 @@ export class CohortEditor extends React.PureComponent<ICohortEditorProps, ICohor
 
     private editFilter(index: number): void {
         const editFilter = this.state.filters[index];
-        this.setState({ isEditingFilter: true, filterIndex: index });
+        this.setState({ filterIndex: index });
         this.setState({ openedFilter: _.cloneDeep(editFilter) });
+        this.setState({ editingFilterIndex: editFilter.column });
```

Comment:
yeah, this is the behavior I'm not clear on. It seems reasonable that edit may be limited to the selected column, but if the user makes a column change, does the UI reflect that they are no longer editing that filter, but creating a new filter (by changing the button labels and any higlight on the selected filter?)

Reply:
This might be too detailed but it might land us with a better implementation 😁

scenario: 
1. the user selects Dataset in the left panel
2. User adds age as a filter , saves filter 
3. User adds education as a filter , saves filter 

use case: 
User now wants to edit age filter. 

behavior:
The UI will show **save** and **cancel** button for age 
![image](https://user-images.githubusercontent.com/16737404/81728815-05e33d00-9459-11ea-9726-6952d5920701.png)

Now user instead changes the filter and selects gender filter. The UI will show **add filter button** and a new filter is created.
![image](https://user-images.githubusercontent.com/16737404/81729032-56f33100-9459-11ea-86a9-807e961d3704.png)

Going back to age will still show save and cancel. 

@rihorn2 - does this make sense or are you thinking of some more visual clues?

Reply:
Based on our discussion - the code should show edit and cancel button when its opened in edit mode. If the user changes column and comes back to the same column as editingFilter it will assumed as adding a new filter.